pkg/config/config.go

```diff
@@ -100,6 +102,8 @@ func InitCliConfig(configAndStacksInfo schema.ConfigAndStacksInfo, processStacks
 	// system dir (`/usr/local/etc/atmos` on Linux, `%LOCALAPPDATA%/atmos` on Windows)
 	// home dir (~/.atmos)
 	// current directory
+	// ENV var ATMOS_CLI_CONFIG_PATH
+	// ENV var ATMOS_REMOTE_CONFIG_URL from GITHUB
```

Comment:
This task should not be implementing a new ENV for fetching remote config. 

- cloudposse/atmos#808

Reply:
@Listener430 lets reduce the scope of this PR. Delete anything that introduces new functionality (like remote atmos configs, new environment variables, etc).